runtime/libffi/z/ffi64.c

```diff
@@ -812,21 +812,24 @@ ffi_check_struct_for_complex(ffi_type *arg_type)
   return arg_type->type;
 }
 
+/*======================== End of Routine ============================*/
+
 /*====================================================================*/
 /*                                                                    */
-/* Name     - ffi_prep_cif_machdep.                                   */
+/* Name     - ffi_prep_cif_machdep_core.                              */
 /*                                                                    */
-/* Function - Perform machine dependent CIF processing.               */
+/* Function - Core implementation of machine dependent CIF processing.*/
```

Comment:
Please adjust this so there's at least one space before */ (perhaps by splitting the line).

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/pull/22396/commits/523147ec00bb79f9113161a74fb9254769aa98f5